challenge-11/challenge-11.js

```diff
@@ -1,54 +1,91 @@
-// Envolva todo o código desse arquivo em uma IIFE (incluindo esse comentário).
+(function(){
+	// Envolva todo o código desse arquivo em uma IIFE (incluindo esse comentário).
 
-/*
-Crie uma variável chamada `once`, que recebe `false` como seu valor.
-Crie um loop que execute enquanto essa variável for verdadeira. Dentro do
-loop, mostre no console a mensagem:
-'Entrou ao menos uma vez!'
-Qual loop você deve usar para que essa mensagem seja mostrada no console?
-*/
-// ?
+	/*
+	Crie uma variável chamada `once`, que recebe `false` como seu valor.
+	Crie um loop que execute enquanto essa variável for verdadeira. Dentro do
+	loop, mostre no console a mensagem:
+	'Entrou ao menos uma vez!'
+	Qual loop você deve usar para que essa mensagem seja mostrada no console?
+	*/
 
-/*
-Crie um objeto chamado `person`, que receba as seguintes propriedades:
-- 'name', 'age', 'weight' e 'birthday'. Preencha com os valores corretos
-para o nome, idade, peso e data de nascimento dessa pessoa.
-*/
-// ?
+	var once = false;
+	do{
+		console.log('Entrou ao menos uma vez');
+		break;
+	} while(once === false)
```

Comment:
-1
Aqui você está dizendo que:

> Enquanto `once` for igual a `false`, execute o loop...

Ou seja, como `once` sempre será `false`, a expressão retorna `true`, e você acaba de criar um loop infinito! xD

O enunciado pede para verificar _"enquanto essa variável for verdadeira"_. Ela nunca será, por isso a mensagem no console só será mostrada uma vez, e você não precisará do `break` para quebrar o fluxo ;)

Deu pra entender bem essa parte?